portal/common/lib/page_fetching.ts

```diff
@@ -70,6 +83,8 @@ export async function fetchPage(
     objectId: string,
     path: string,
 ): Promise<Response> {
+    // TODO: On the portal side, fetch the route object, store it to a routes object.
```

Comment:
Question: is the todo still valid? if it is, link it to an issue.

Reply:
No, that is already done ✅

Reply:
Removed.